### PR TITLE
[No QA] Ask for affected user email in standard issue template

### PR DESCRIPTION
**Pull request by marcaaron**
### Fixed Issues
$ https://github.com/Expensify/App/issues/7816

### Tests
None

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Standard.md [hunk 1]
```diff
@@ -34,10 +34,11 @@ Where is this issue occurring?
 **Version Number:**
 **Reproducible in staging?:**
 **Reproducible in production?:**
+**Email of affected user:**
 **Logs:** https://stackoverflow.com/c/expensify/questions/4856
 **Notes/Photos/Videos:** Any additional supporting documentation
 **Expensify/Expensify Issue URL:**
 **Issue reported by:**
-**Slack conversation:**
```
Comment (Beamanator) [inline .github/ISSUE_TEMPLATE/Standard.md:41]: NAB; Personally I kinda like these extra spaces after these so you don't have to add them when filling out the template 🤷

File: .github/ISSUE_TEMPLATE/Standard.md [hunk 2]
```diff
@@ -34,10 +34,11 @@ Where is this issue occurring?
 **Version Number:**
 **Reproducible in staging?:**
 **Reproducible in production?:**
+**Email of affected user:**
```
Comment (Beamanator) [inline .github/ISSUE_TEMPLATE/Standard.md:37]: NAB; Think we should say Email / phone here in case their primary login is a phone number?

**Review by Beamanator — DISMISSED** [review]
Two little NABs, feel free to address or not 😅

File: .github/ISSUE_TEMPLATE/Standard.md (hunk 1 above)
Comment (marcaaron) [reply .github/ISSUE_TEMPLATE/Standard.md:41]: I think I just have auto-trim whitespace set so it's not possible to save a file without getting rid of the whitespace lol I didn't think about how they might be useful.

**Comment by marcaaron** [comment]
Updated thanks for the review @Beamanator !

**Review by Beamanator — APPROVED** [review]

**Pull request merged**